go.mod

```
@@ -4,23 +4,22 @@ go 1.13
 
 require (
 	github.com/ahmetb/gen-crd-api-reference-docs v0.1.5
-	github.com/gardener/gardener v1.2.1-0.20200408030154-40b97d31d7f7
-	github.com/gardener/gardener-extensions v1.6.1
+	github.com/gardener/gardener v1.6.0
```

maybe we can vendor latest? 

```suggestion
	github.com/gardener/gardener v1.6.5
```